models/backup_capture.py

```
@@ -0,0 +1,389 @@
+                conn.executemany(
+                    'INSERT OR IGNORE INTO backup_capture_key (capture_id, slot_key) VALUES (?, ?)',
+                    [(capture_id, key) for key in dict.fromkeys(slot_keys)],
+                )
+            conn.commit()
+        return capture_id
+
+    def update(self, capture_id: str, updates: Dict) -> bool:
+        if not updates:
+            return False
+        clause = ', '.join(f"{k} = ?" for k in updates)
+        values = list(updates.values()) + [capture_id]
```

Ruff (0.16.0)

[warning] Consider `[*list(updates.values()), capture_id]` instead of concatenation

Replace with `[*list(updates.values()), capture_id]`

(RUF005)